Allphi/Domain/Domaincontroller.cs

```diff
@@ -73,17 +74,127 @@ public ParkingSpot GetAvailableParkingSpotVisitor()
             return (_parkingRepo.GetAvailableParkingSpotUnreserved());
         }
 
-        #endregion Parking
-
-        public void SubmitVisitor(string licensePlate)
+        public void SubmitVisitorParking(string licensePlate)
         {
             string license = Guard.Against.NullOrEmpty(licensePlate, nameof(licensePlate));
-            ParkingSpot parkingSpot = GetAvailableParkingSpotVisitor();
+            ParkingSpot parkingSpot = _parkingRepo.GetAvailableParkingSpotUnreserved();
             if (IsLicensePlateValid(license) && parkingSpot != null && _parkingRepo.CountParkingSpotByPlate(license) == 0)
             {
                 parkingSpot.Plate = license;
                 _parkingRepo.UpdateParkingSpot(parkingSpot);
             };
         }
+
+        public void SubmitEmployeeParking(string licensePlate)
+        {
+            string license = Guard.Against.NullOrEmpty(licensePlate, nameof(licensePlate));
+            Employee employee = _employeeRepo.GetEmployeeByPlate(licensePlate);
+            ParkingSpot parkingSpot = _parkingRepo.GetAvailableParkingSpotReserved(employee.Business);
+            if (parkingSpot == null) parkingSpot = _parkingRepo.GetAvailableParkingSpotUnreserved();
+            if (IsLicensePlateValid(licensePlate) && parkingSpot != null &&
+                _parkingRepo.CountParkingSpotByPlate(license) == 0)
+            {
+                parkingSpot.Plate = license;
+                _parkingRepo.UpdateParkingSpot(parkingSpot);
+            }
+        }
+
+        #endregion Parking
+
+        #region Business
+
+        public void CreateBusiness(string name, string? address, string? phone, string email, string btw)
+        {
+            _businessRepo.CreateBusiness(new Business(name, btw, email, address, phone));
+        }
+
+        public List<Business> GetBusinesses()
+        {
+            return _businessRepo.GetBusinesses();
+        }
+
+        #endregion Business
+
+        public List<Visitor> GetVisitors()
+        {
+            return _visitorRepo.GetVisitors();
+        }
+
+        public List<Contract> GetContracts()
+        {
+            return _contractRepo.GetContracts();
+        }
+
+        public List<Employee> GetEmployees()
+
+        {
+            return _employeeRepo.GetEmployees();
+        }
+
+        public List<string> ConvertBusinessToStringList(object input)
```

Comment:
Putting the return type in a function name (or the type name of a variable for that matter) is bad practise and serves only to make it harder to read.
I can already see it returns `List<string>`, putting ToStringList is redundant.